config/core/configmaps/features.yaml

```diff
@@ -109,6 +109,7 @@ data:
     # - RunAsNonRoot
     # - SupplementalGroups
     # - RunAsUser
+    # - SeccompProfile
```

Comment:
Just leaving a note as a FYI:
This yaml file is not used but https://github.com/openshift-knative/serving/blob/release-v1.7/openshift/release/artifacts/2-serving-core.yaml#L5000-L5006 is used on 1.7 branch (release-next & main fixed).

This is a comment line change so I think we don't need to update it, though.

Reply:
Hm I see, was it used in 1.6 (https://github.com/openshift-knative/serving/pull/9/files)? Maybe @skonto just added that for completeness sake?

Reply:
Yes that was meant for completeness back then.